pkg/netpol/connlist/conns_formatter.go

```diff
@@ -126,3 +136,29 @@ func (d dotFormatter) writeOutput(conns []Peer2PeerConnection) (string, error) {
 	allLines = append(allLines, dotClosing)
 	return strings.Join(allLines, getNewLineChar()), nil
 }
+
+// csvFormatter: implements the connsFormatter interface for csv output format
+type csvFormatter struct {
```

Comment:
Since `CSV` is an acronym *and* there is no intention to export the struct, may want to rename it as `formatCSV`.
Similarly for other formats.